LeVent.Infrastructure.Build/Program.cs

```diff
@@ -1,6 +1,6 @@
-﻿// -------------------------------------------------
-// Copyright (c) PiorSoft, LLC. All rights reserved.
-// -------------------------------------------------
+﻿// ----------------------------------------------------------------------------------
```

Comment:
Can you shorten the lines above and below to align with the text in the middle?

Reply:
@cjdutoit Fixed